extensions/cornerstone/src/utils/measurementServiceMappings/Length.ts

```diff
@@ -18,9 +21,16 @@ const Length = {
     getValueTypeFromToolType,
     customizationService
   ) => {
-    const { annotation, viewportId } = csToolsEventDetail;
+    const { annotation } = csToolsEventDetail;
     const { metadata, data, annotationUID } = annotation;
 
+    const isLocked = getIsLocked(annotationUID);
+    const isVisible = getIsVisible(annotationUID);
+    const colorString = config.style.getStyleProperty('color', { annotationUID });
+
+    // color string is like 'rgb(255, 255, 255)' we need them to be in RGBA array [255, 255, 255, 255]
+    const color = colorString.replace('rgb(', '').replace(')', '').split(',').map(Number);
```

Comment:
Even though this is trivial, could you define this in a utility?  It just feels like something which will be needed elsewhere.

Reply:
It is not implemented so i'm removing it for now